blockchain/blockchain-wallet-v4-frontend
# fix(Remind GUID): wait until request submission is successful

## Description
1069

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
1. go to remind guid
2. enter email and incorrect captcha
3. see error
4. enter correct captcha
5. see success and go to next page
6. test form is reset

## Code Checklist
